### DISCUSSION TITLE
liquibase parameters

### DISCUSSION THREAD

**Discussion by luulac2022** [quarkusio/quarkus#34036] [Community]
Hi guys, I want to ask some question from liquibase in quarkus.
The first question is: "How to I configuration this parameter in quarkus?"

I found that in document of liquibase:

```
--validate-xml-changelog-files
liquibase.validateXmlChangelogFiles

If true, will perform XSD validation of XML changelog files. When many XML changelog files are included, this validation may impact Liquibase performance. Default: true. See validate-xml-changelog-files.
```

But in quarkus not have any idea about it.

Thank guys.

**Comment by kahennig** [comment]
hi @luulac2022, 
did you ever find a way to do this?
Thanks
